### ISSUE TITLE
Allow EC commands to work with `/execute as`

### ISSUE THREAD

**Issue by John-Paul-R** [John-Paul-R/Essential-Commands#139]
> hey, I'd like to propose a feature:

> That being that these commands should be executable by blocks.
> For example: I should be able to click on a sign that has the rtp clickevent, and it should execute it. or > let a cmd block execute it for me:

```
execute as @a[tag=rtp] run rtp
```

> Because currently the cmd block output simply gives an error msg for whatever reason
> Or make sth fancy like this:

```js
minecraft:dark_oak_sign{BlockEntityTag:{Text1:'{"text":"Click Here To","clickEvent":{"action":"run_command","value":"rtp"},"color":"gray"}',Text2:'{"text":"Explore The","color":"gray"}',Text3:'{"text":"Wilderness","color":"green"}'},display:{Name:'{"text":"RTP"}'}}
```

> and commands like /spawn or similar.
> Or give temporary nicks utilizing cmd blocks

https://discord.com/channels/513219024927326211/857077956336943154/1006362543649992764

**Comment by John-Paul-R** [comment]
So... this just seems to work out of the box on `0.28.1-mc1.19`

**Comment by John-Paul-R** [comment]
this was an issue on older versions of EC (in this case, `0.20.4-mc1.18.1`). This has been resolved for 1.18.1 in [0.24.5-mc1.18.1](https://github.com/John-Paul-R/Essential-Commands/releases/tag/0.24.5-mc1.18.1) (backport).
